pages/views/core/esl-utils.njk

```diff
@@ -4,7 +4,7 @@ title: ESL Utils
 seoTitle: Universal JS, TS and Web API utilities set from Exadel Smart Library
 name: ESL Utils
 tags: core
-iconAlt: '@bind @memoize @prop'
+icon: 'common/right-arrow.svg'
```

Comment:
It looks like we don't need to set this information to pages directly, it is common styling